src/surfing_penguin/routes.py

```diff
@@ -50,6 +51,18 @@ def post(self):
         return "Login: %s, %s" % (user, password)
 
 
+@api.route('/api_register')
+class api_register(Resource):
+    def get(self):
+        return "Register Page"
+
+    def post(self):
+        username = request.form['username']
+        password = request.form['password']
+        user = User(username, password)
+        return "user: %s, passwd:%s" % (user.username, user.password)
+
```

Comment:
You need to provide a test for this function, either a simple html or unit test.